client/service/dashboard_client.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=MHP DAShboard client
+
+[Install]
+WantedBy=default.target
+
+[Service]
+Environment=NODE_PORT=3000
+Restart=on-failure
+# Note for this to work we need to run the below command from the DAShboard repository client directory, the Working directory is set by install.sh script
+# Ensure serve is downloaded: `yarn global add serve`
+ExecStart=/home/pi/.yarn/bin/serve -s build
```

Comment:
You could generate this line in the `install.sh` using `yarn global bin` so that we don't rely on the user doing it

Reply:
I tried a few ways of doing it but with no avail. command substitution (i.e `$(yarn global bin)`) doesn't work with the simply command scanner that systemd uses. I also tried making `install.sh` add an environment variable to the `service` file with the path to `serve` (since `install.sh` can run command substitution). And I have also tried making `install.sh` add the path `yarn global bin` to `$PATH` but it appears systemd doesn't use that var to find executables.

There probably is a better way, but I decided no to waste more time into it. However, if you have any solution in mind let me know and I can try it.